implementations/rust/ockam/ockam_command/src/reset.rs

```diff
@@ -11,33 +13,32 @@ pub struct ResetCommand {
 
 impl ResetCommand {
     pub fn run(self, opts: CommandGlobalOpts) {
-        if self.yes || get_user_confirmation() {
-            if let Err(e) = run_impl(opts) {
-                eprintln!("{e}");
-                std::process::exit(e.code());
-            }
+        if let Err(e) = run_impl(opts, self) {
+            eprintln!("{e}");
+            std::process::exit(e.code());
         }
     }
 }
 
-fn run_impl(opts: CommandGlobalOpts) -> crate::Result<()> {
+fn run_impl(opts: CommandGlobalOpts, cmd: ResetCommand) -> crate::Result<()> {
+    match opts
+        .terminal
+        .confirm("This will delete the local CLI state. Are you sure?")?
```

Comment:
We need a better message here. CLI state has no meaning to end users.

Reply:
I changed it to "local Ockam configuration". The point I want to make is that the command only affects local stuff.

Reply:
@mrinalwadhwa are you ok with that wording? Can we merge?

Reply:
That change is good but there is another reference to CLI state in this PR

Reply:
Right, in the last commit I changed the wording everywhere CLI state was used

Reply:
Thanks @etorreborre ! I didn't see the other reference and forgot to follow up on this